src/managers/queue-manager.js

```diff
@@ -20,7 +20,7 @@ export async function startQueueListener() {
     if (getUserToken()) {
       log("Queue watcher started");
       pollingSetup = true;
-      poll(() => new Promise(() => pollMessageQueue()), POLLING_DELAY_IN_SECONDS);
+      poll(() => new Promise(() => pullMessagesFromQueue()), 1000);
```

Comment:
Did this just change from 10s to 1s?

Reply:
We're checking the local store every second to see if we can check the queue.

Reply:
I would suggest using a constant here to define the DELAY